moped-editor/src/index.jsx

```diff
@@ -5,40 +5,18 @@ import Amplify from "aws-amplify";
 import React from "react";
 import ReactDOM from "react-dom";
 import "./_index.scss";
-import config from "./config";
 import * as serviceWorker from "./serviceWorker";
 import App from "./app/App";
 
 // cssVars();
 
-/**
- * AWS Amplify
- * @see https://github.com/aws-amplify/amplify-js
- */
-
-Amplify.Logger.LOG_LEVEL = 'DEBUG';
-
-Amplify.configure({
-    Auth: {
-        mandatorySignIn: true,
-        region: config.cognito.REGION,
-        userPoolId: config.cognito.USER_POOL_ID,
-        identityPoolId: config.cognito.IDENTITY_POOL_ID,
-        userPoolWebClientId: config.cognito.APP_CLIENT_ID
-    },
-    API: {
-        endpoints: [
-            {
-                name: "testApi",
-                endpoint: config.apiGateway.URL,
-                region: config.apiGateway.REGION
-            }
-        ]
-    }
-});
-
-
-ReactDOM.render(<App />, document.getElementById("root"));
```

Comment:
We really need the Amplify library here, sorry things are confusing right now, but let's put this block of code back in your branch whenever you get a chance.